Library/Formula/natalie.rb

```diff
@@ -0,0 +1,23 @@
+class Natalie < Formula
+  homepage "https://github.com/krzyzanowskim/Natalie"
+  url "https://github.com/krzyzanowskim/Natalie/archive/0.1.tar.gz"
+  sha256 "943dab1b7ac16555ec3047983a3c397f7ebe7559d3c098c02a6ee6d8e966655e"
+  head "https://github.com/krzyzanowskim/Natalie.git"
+
+  depends_on :xcode => "6.0"
+
+  def install
+    bin.install "natalie.swift"
+    etc.install Dir["NatalieExample"] unless File.exists? (etc/"NatalieExample")
```

Comment:
@jacknagel Done on c13cc05373292d0d7ef1a402646b10a8b3668eb9! Thanks for pointing out :smiley: